### PR TITLE
Release 1.118.0

### PR DESCRIPTION
**Pull request by twstokes**
Release for Gutenberg Mobile 1.118.0

## Related PRs

- https://github.com/WordPress/gutenberg/pull/61103
- https://github.com/wordpress-mobile/WordPress-Android/pull/20719
- https://github.com/wordpress-mobile/WordPress-iOS/pull/23092

## Changes

- https://github.com/wordpress-mobile/gutenberg-mobile/pull/6716
- https://github.com/wordpress-mobile/gutenberg-mobile/pull/6804

## Test plan

Once the installable builds of the main apps are ready, perform a quick smoke test of the editor on both iOS and Android to verify it launches without crashing. We will perform additional testing after the main apps cut their releases.

## Release Submission Checklist

- [ ] Verify Items from test plan have been completed
- [x] Check if `RELEASE-NOTES.txt` is updated with all the changes that made it to the release. Replace `Unreleased` section with the release version and create a new `Unreleased` section.
- [x] Check if `gutenberg/packages/react-native-editor/CHANGELOG.md` is updated with all the changes that made it to the release. Replace `## Unreleased` with the release version and create a new `## Unreleased`.
- [x] Bundle package of the release is updated.

### CODE REVIEW

File: bundle/android/strings.xml [hunk 1]
```diff
@@ -227,9 +227,9 @@
     <string name="gutenberg_native_jetpack_powered" tools:ignore="UnusedResources">Jetpack powered</string>
     <string name="gutenberg_native_line_height" tools:ignore="UnusedResources">Line Height</string>
     <string name="gutenberg_native_link_inserted" tools:ignore="UnusedResources">Link inserted</string>
+    <string name="gutenberg_native_link_label" tools:ignore="UnusedResources">Link label</string>
```
Comment (geriux) [inline bundle/android/strings.xml:230]: These changes are related to https://github.com/WordPress/gutenberg/pull/60047 so it's ok they're being updated here.

File: bundle/ios/GutenbergNativeTranslations.swift [hunk 2]
```diff
@@ -208,9 +208,9 @@ private func dummy() {
     _ = NSLocalizedString("Jetpack powered", comment: "")
     _ = NSLocalizedString("Line Height", comment: "")
     _ = NSLocalizedString("Link inserted", comment: "")
+    _ = NSLocalizedString("Link label", comment: "")
```
Comment (geriux) [inline bundle/ios/GutenbergNativeTranslations.swift:211]: Same as https://github.com/wordpress-mobile/gutenberg-mobile/pull/6832/files#r1579907650

**Review by geriux — APPROVED** [review]
LGTM! 🚀

**Comment by geriux** [comment]
## 🟢 Basic Test Suite Checklist (Android)
**Build**: [24.8-rc-1](https://github.com/wordpress-mobile/WordPress-Android/releases/tag/24.8-rc-1)
**Device:** Samsung Galaxy A23 5G (Android 11)

### Writing Flow

#### Rich Text Format
- [x] [TC007](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/rich-text-formatting.md#tc007) - Test format detection under the cursor
- [x] [TC009](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/rich-text-formatting.md#tc009) - Test autocorrection doesn't apply formatting to Heading
- [x] [TC010](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/rich-text-formatting.md#tc010) - Test autocorrection doesn't remove formatting from Heading
##### Splitting and merging
- [x] [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/splitting-merging.md#tc001) - Merge after writing
  - [x] Paragraph
  - [x] Heading
- [x] [TC002](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/splitting-merging.md#tc002) - Merge after selection
  - [x] Paragraph
  - [x] Heading
⚠️ Encountered the known issue with GBoard https://github.com/wordpress-mobile/gutenberg-mobile/issues/2375
- [x] [TC003](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/splitting-merging.md#tc003) - Merge after deleting text
  - [x] Paragraph
  - [x] Heading
- [x] [TC004](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/splitting-merging.md#tc004) - Merge after deleting all
  - [x] Paragraph
  - [x] Heading
- [x] [TC005](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/splitting-merging.md#tc005) - Merge multiple blocks
  - [x] Paragraph
  - [x] Heading


### Unsupported Block
- [x] [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/unsupported-block-editing.md#tc001) - User can edit unsupported blocks on Simple WP.com sites
- [x] [TC002](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/unsupported-block-editing.md#tc002) - User can discard edits to an unsupported block on Simple WP.com sites
- [x] [TC003](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/unsupported-block-editing.md#tc003) - Editing unsupported blocks is allowed on Gutenberg-enabled Atomic sites
- [x] [TC004](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/unsupported-block-editing.md#tc004) - Editing unsupported blocks is disallowed on Classic-enabled Atomic sites
- [x] [TC005](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/unsupported-block-editing.md#tc005) - Editing unsupported blocks is enabled on self-hosted sites accessed via Jetpack
- [x] [TC006](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/unsupported-block-editing.md#tc006) - Editing unsupported blocks is disallowed on self-hosted sites access via their own username and password

### Gallery Block
- [x] [TC005](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/gallery.md#tc005) - Choose from device (stay in editor) - Successful upload

**Comment by geriux** [comment]
## 🟢 New changes (Android)
**Build**: [24.8-rc-1](https://github.com/wordpress-mobile/WordPress-Android/releases/tag/24.8-rc-1)
**Device:** Samsung Galaxy A23 5G (Android 11)

- [ ] https://github.com/wordpress-mobile/gutenberg-mobile/pull/6716 **(iOS Only)**
- [x] https://github.com/wordpress-mobile/gutenberg-mobile/pull/6804

**Comment by geriux** [comment]
## 🟢 Functionality tests - Test Suite 3 (Android)
**Build**: [24.8-rc-1](https://github.com/wordpress-mobile/WordPress-Android/releases/tag/24.8-rc-1)
**Device:** Samsung Galaxy A23 5G (Android 11)

### Gallery - 2
- [x] Gallery block - Insert image from device (failing) - [TC006](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/gallery.md#tc006)
- [x] Gallery block - Insert image from device (cancel) - [TC009](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/gallery.md#tc009)

### Gallery - 3
- [x] Gallery block - Settings: Column number - [TC013](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/gallery.md#tc013)
- [x] Gallery block - Settings: Crop images - [TC014](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/gallery.md#tc014)

### MediaText - 3
- [x] Media Text block - Insert video from device (failing) - [TC001-v](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/media-text.md#tc001-v)
- [x] Media Text block - Close/Re-open post with an ongoing video upload - [TC003-v](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/media-text.md#tc003-v)

### MediaText - 4
- [x] Media Text block - Media & Text alignment - [TC006](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/media-text.md#tc006)
- [x] Media Text block - Vertical alignment - [TC007](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/media-text.md#tc007)

### Shortcode-1
- [x] Shortcode block - Add a youtube link - [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/shortcode.md#tc001)

### Cover - 2

- [x] Cover - Add Video Background from WordPress Media library - [TC004](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/cover.md#tc004)
- [x] Cover - Replace media - [TC005](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/cover.md#tc005)
- [x] Cover - Gradient background is rendered properly - [TC006](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/cover.md#tc006)
- [x] Cover - Gradient overlay is rendered properly - [TC007](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/cover.md#tc007)

### Image - 2
- [x] Image block - Border style setting - [TC006](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/image.md#tc006)
- [x] Image block - Image size setting - [TC007](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/image.md#tc007)
- [x] Image block - Link to setting - [TC008](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/image.md#tc008)

**Comment by twstokes** [comment]
## 🟢 Functionality tests - Test suite 4 (iOS)

**Build:** TestFlight 24.8.0.2
**Device:** iPhone 12 (iOS 17.4.1)

### Group - 2

- [x] Group - Check if nested Placeholder block can be replaced - [TC010](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/group.md#tc010)

### Buttons - 4

- [x] Buttons block - The newly created Buttons has set background color to the theme-default color in preview - [TC017](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/buttons.md#tc017)

### Buttons - 5

- [x] Buttons block - Render theme colors - [TC015](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/buttons.md#tc015)
- [x] Buttons block - Settings: Change Button border radius - [TC011](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/buttons.md#tc011)

### Buttons - 7

- [x] Buttons block - Toolbar link button is active when Button has link - [TC018](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/buttons.md#tc018)

### Editor Theme - 2
- [x] Custom Gradients - Check that custom gradients load in the editor - [TC004](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/editor-theme.md#tc004)
- [x] Offline Support - [TC005](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/editor-theme.md#tc005)

### Color Settings - 2
- [x] Customize gradient color - [TC005](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/color-settings.md#tc005)
- [x] Custom color picker - [TC006](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/color-settings.md#tc006)

### VideoPress - 1
- [x] VideoPress block - Upload a video - [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/jetpack/videopress.md#tc001)
- [x] VideoPress block - Close post with an ongoing video upload - [TC004](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/jetpack/videopress.md#tc004)
  - ⚠️ No green progress bar was shown in the post list, but this worked great otherwise.

**Comment by twstokes** [comment]
## 🟢 New changes (iOS)
**Build:** TestFlight 24.8.0.2
**Device:** iPhone 12 (iOS 17.4.1)

- [x] https://github.com/wordpress-mobile/gutenberg-mobile/pull/6716 **(iOS Only)**
- [x] https://github.com/wordpress-mobile/gutenberg-mobile/pull/6804

**Comment by twstokes** [comment]
## 🟢 Basic Test Suite Checklist (iOS)
**Build:** TestFlight 24.8.0.2
**Device:** iPhone 12 (iOS 17.4.1)

### Writing Flow

#### Rich Text Format
- [x] [TC007](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/rich-text-formatting.md#tc007) - Test format detection under the cursor
- [x] [TC009](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/rich-text-formatting.md#tc009) - Test autocorrection doesn't apply formatting to Heading
- [x] [TC010](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/rich-text-formatting.md#tc010) - Test autocorrection doesn't remove formatting from Heading
##### Splitting and merging
- [x] [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/splitting-merging.md#tc001) - Merge after writing
  - [x] Paragraph
  - [x] Heading
- [x] [TC002](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/splitting-merging.md#tc002) - Merge after selection
  - [x] Paragraph
  - [x] Heading
- [x] [TC003](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/splitting-merging.md#tc003) - Merge after deleting text
  - [x] Paragraph
  - [x] Heading
- [x] [TC004](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/splitting-merging.md#tc004) - Merge after deleting all
  - [x] Paragraph
  - [x] Heading
- [x] [TC005](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/splitting-merging.md#tc005) - Merge multiple blocks
  - [x] Paragraph
  - [x] Heading


### Unsupported Block
- [x] [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/unsupported-block-editing.md#tc001) - User can edit unsupported blocks on Simple WP.com sites
- [x] [TC002](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/unsupported-block-editing.md#tc002) - User can discard edits to an unsupported block on Simple WP.com sites
- [x] [TC003](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/unsupported-block-editing.md#tc003) - Editing unsupported blocks is allowed on Gutenberg-enabled Atomic sites
- [x] [TC004](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/unsupported-block-editing.md#tc004) - Editing unsupported blocks is disallowed on Classic-enabled Atomic sites
- [x] [TC005](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/unsupported-block-editing.md#tc005) - Editing unsupported blocks is enabled on self-hosted sites accessed via Jetpack
- [x] [TC006](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/unsupported-block-editing.md#tc006) - Editing unsupported blocks is disallowed on self-hosted sites access via their own username and password

### Gallery Block
- [x] [TC005](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/gallery.md#tc005) - Choose from device (stay in editor) - Successful upload

**Pull request merged**